components/CustomButton/CustomButton.tsx

```diff
@@ -119,7 +119,7 @@ const CustomButton: React.FC<CustomButtonProps> = ({
       {...props}
       buttonLabel={buttonLabel}
     >
-      <Typography>{buttonLabel}</Typography>
+      {buttonLabel}
```

Comment:
Why did you remove Typography?

Reply:
I define the button fontsize in the custom button component but wrapping the button label with a typography tag again applies the typography fontsizes on the button over the other font sizes